nodejs/eks/cni/aws-k8s-cni.yaml

```diff
@@ -60,143 +68,152 @@ metadata:
-            - mountPath: /var/run/dockershim.sock
-              name: dockershim
-            - mountPath: /run/xtables.lock
-              name: xtables-lock
-      hostNetwork: true
-      initContainers:
-        - env: []
-          image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni-init:v1.7.5
-          imagePullPolicy: Always
-          name: aws-vpc-cni-init
-          securityContext:
-            privileged: true
-          volumeMounts:
-            - mountPath: /host/opt/cni/bin
-              name: cni-bin-dir
-      priorityClassName: system-node-critical
-      serviceAccountName: aws-node
-      terminationGracePeriodSeconds: 10
-      tolerations:
-        - operator: Exists
+          - mountPath: /host/opt/cni/bin
+            name: cni-bin-dir
+          - mountPath: /host/etc/cni/net.d
+            name: cni-net-dir
+          - mountPath: /host/var/log/aws-routed-eni
+            name: log-dir
+          - mountPath: /var/run/dockershim.sock
+            name: dockershim
+          - mountPath: /var/run/aws-node
+            name: run-dir
+          - mountPath: /run/xtables.lock
+            name: xtables-lock
       volumes:
-        - hostPath:
-            path: /opt/cni/bin
-          name: cni-bin-dir
-        - hostPath:
-            path: /etc/cni/net.d
-          name: cni-net-dir
-        - hostPath:
-            path: /var/run/dockershim.sock
-          name: dockershim
-        - hostPath:
-            path: /run/xtables.lock
-          name: xtables-lock
-        - hostPath:
-            path: /var/log/aws-routed-eni
-            type: DirectoryOrCreate
-          name: log-dir
-        - hostPath:
-            path: /var/run/aws-node
-            type: DirectoryOrCreate
-          name: run-dir
-
+      - name: cni-bin-dir
+        hostPath:
+          path: /opt/cni/bin
+      - name: cni-net-dir
+        hostPath:
+          path: /etc/cni/net.d
+      - name: dockershim
+        hostPath:
+          path: /var/run/dockershim.sock
+      - name: log-dir
+        hostPath:
+          path: /var/log/aws-routed-eni
+          type: DirectoryOrCreate
+      - name: run-dir
+        hostPath:
+          path: /var/run/aws-node
+          type: DirectoryOrCreate
+      - name: xtables-lock
+        hostPath:
+          path: /run/xtables.lock
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: kubernetes.io/os
+                operator: In
+                values:
+                - linux
+              - key: kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
+                - arm64
+              - key: eks.amazonaws.com/compute-type
+                operator: NotIn
+                values:
+                - fargate
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Could you add a changelog entry to call out the change and in particular the bump of the api version for the eniconfigs custom resource?

Something along this lines would be great:
```
- Add `enableIPv6` to VPC CNI options
- **BREAKING** Update API version for `eniconfigs.crd.k8s.amazonaws.com` CRD to  `apiextensions.k8s.io/v1` since `apiextensions.k8s.io/v1beta1` support was dropped in Kubernetes 1.22
```

Reply:
done.